src/window_internal_glutin.rs

```diff
@@ -355,6 +355,8 @@ impl<UserEventType: 'static> WindowGlutin<UserEventType>
         // Show window after positioning to avoid the window jumping around
         window_context.window().set_visible(true);
 
+        window_context.window().set_cursor_hittest(!options.mouse_passthrough).unwrap();
```

Comment:
Thanks for this PR, it mostly looks good! Currently this will crash on platforms where this isn't supported -- might be better to add a new `WindowCreationError` instead and return that?